client/app/iplb/frontends/iplb-frontends-edit.html

```diff
@@ -0,0 +1,150 @@
+          </div>
+        </div>
+
+        <div class="oui-select-formfield" data-ng-if="ctrl.frontend.ssl">
+          <div class="oui-select">
+            <cui-loader data-ng-show="ctrl.certificates.loading"></cui-loader>
+            <select class="oui-select__input"
+                name="certificate"
+                id="certificate"
+                data-ng-hide="ctrl.certificates.loading"
+                data-ng-model="ctrl.frontend.defaultSslId"
+                data-ng-options="certificate.id as ctrl.getCertificateName(certificate) for certificate in ctrl.certificates.data">
+            </select>
+            <label for="certificate" class="oui-label" data-translate="iplb_frontend_add_default_certificate"></label>
+          </div>
+        </div>
+
+        <cui-advanced-options data-title="'iplb_frontend_add_advanced_options' | translate"
+            data-text="'iplb_frontend_add_advanced_options_info' | translate"
+            data-button-text="'iplb_frontend_add_advanced_options_show' | translate"
+            data-expanded="ctrl.edition">
+            <div class="oui-select-formfield">
+                <div class="oui-select">
+                    <cui-loader data-ng-show="ctrl.dedicatedIp.loading"></cui-loader>
+                    <select class="oui-select__input"
+                        name="dedicatedIpfo"
+                        id="dedicatedIpfo"
+                        data-ng-model="ctrl.frontend.dedicatedIpfo"></select>
+                    <label for="dedicatedIpfo" class="oui-label" data-translate="iplb_frontend_add_dedicated_ip_failover"></label>
+                </div>
+            </div>
+
+            <div class="oui-input-formfield" data-ng-if="ctrl.type != 'udp'">
+              <input type="text" id="restrict_ip_access" name="restrict_ip_access" class="oui-input"
+                 data-ng-model="ctrl.frontend.allowedSource" />
+              <label for="restrict_ip_access" class="oui-label" data-translate="iplb_frontend_add_restrict_ip_access"></label>
+            </div>
+
+            <div class="oui-input-formfield" data-ng-if="ctrl.type == 'http'">
+              <input type="text" id="http_redirection" name="http_redirection" class="oui-input"
+                 data-ng-model="ctrl.frontend.redirectLocation" />
+              <label for="http_redirection" class="oui-label" data-translate="iplb_frontend_add_http_redirection"></label>
+            </div>
+
+            <div class="oui-input-formfield" data-ng-if="ctrl.type == 'http'">
+              <input type="text" id="http_header" name="http_header" class="oui-input"
+                 data-ng-model="ctrl.frontend.httpHeader" />
+              <label for="http_header" class="oui-label" data-translate="iplb_frontend_add_http_header"></label>
+            </div>
+
+            <div class="oui-checkbox" data-ng-if="ctrl.type == 'http' && ctrl.frontend.ssl">
```

Comment:
These is a component oui-checkbox in oui-angular that you might prefer from this one.